contrib/charts/navigator/templates/rbac.yaml

```diff
@@ -87,7 +87,7 @@ items:
     name: "{{ template "fullname" . }}:controller"
   rules:
   - apiGroups: ["navigator.jetstack.io"]
-    resources: ["elasticsearchclusters", "pilots"]
+    resources: ["elasticsearchclusters", "pilots", "elasticsearchclusters/status", "pilots/status"]
```

Comment:
❓ Does a controller ever need to modify `pilots/status` ?

Reply:
It needs to delegate permission to modify/update `pilots/status` to each Pilot.

Reply:
Got it.